clients/python/sysadmin/Migrations.py

```diff
@@ -28,7 +28,7 @@ def load_migrations(path):
 
 
 def hash_migration(migration):
-    return hashlib.md5(yaml.dump(migration)).hexdigest()
+    return hashlib.md5(yaml.dump(migration, default_flow_style=None)).hexdigest()
```

Comment:
according to the documentation this parameter is suppose to be a boolean:

"If you want collections to be always serialized in the block style, set the parameter default_flow_style of dump() to False."

https://pyyaml.org/wiki/PyYAMLDocumentation

Reply:
Yeah the documentation kinda sucks, took me a while to dig this up. There is different behavior for `None`, `False`, & `True`. Only `None` matches the previous behavior.
```python
>>> yaml.dump(migration_one, default_flow_style=None)
'default: &id001\n  radio0: {agg_incremental: 8, agg_max: 16}\nnew:\n  wireless: *id001\n'
>>> yaml.dump(migration_one, default_flow_style=False)
'default: &id001\n  radio0:\n    agg_incremental: 8\n    agg_max: 16\nnew:\n  wireless: *id001\n'
>>> yaml.dump(migration_one, default_flow_style=True)
'{default: &id001 {radio0: {agg_incremental: 8, agg_max: 16}}, new: {wireless: *id001}}\n'
```

on qbert (pyyaml=3.12):
```python
>>> yaml.dump(migration_one)
'default: &id001\n  radio0: {agg_incremental: 8, agg_max: 16}\nnew:\n  wireless: *id001\n'
```

Reply:
that is nasty, good find